predictoor/pdr-parameters.md

```diff
@@ -0,0 +1,23 @@
+# Predictoor Parameters
+
+## Feeds Published
+For testnet, there are 10 feeds: X/USDT pair for each of the top-10 coins by market cap (ignoring stablecoins), 5m timescales, on Binance, >0% fees on Binance. Paid feeds. The coins are: X = BTC, ETH, BNB, XRP, ADA, DOGE, SOL, LTC, TRX, DOT
+
+For mainnet, tentatively, there will be 20 feeds:  
+- 10x at 5m timescales like above  
+- 10x at 60m timescales like above  
+
+## Pricing
```

Comment:
Below "Pricing" header and "The price..." paragraph, add:

"For each timescale, one feed is free: BTC/UTD on Binance. Below is pricing for the remaining feeds."

Why: it's useful to report the free pricing. FYI the blog post is also updated accordingly.

Reply:
Done